transpiler/__tests__/target/typescript/core/domainError.feature

```diff
@@ -0,0 +1,20 @@
+ # https://docs.google.com/spreadsheets/d/1Rz8m8LxJwXfBKGCcQzPqnry3WiWmrdDDy_vgM4whYwM/edit#gid=0
```

Comment:
We already have a test for domainErrors  `transpiler/__tests__/target/typescript/core/domainErrors.feature`, but we've made some changes there too. In the end this test should be removed in order not to have duplicates.